dealership/contracts.py

```diff
@@ -1,12 +1,46 @@
+
 class Contract(object):
-    pass
+  def __init__(self, vehicle, customer):
+    self.vehicle = vehicle 
+    self.customer = customer
+    self.total = 0
+  
+  def total_value(self):
+    if self.customer.is_employee(): 
+       self.total=  self.total * 0.9
+    return self.total
+  
+  def monthly_value(self):
+    self.total = self.total_value()
+    monthly_value = self.total / self.MONTH
+    return monthly_value
+      
 
 
 class BuyContract(Contract):
     def __init__(self, vehicle, customer, monthly_payments):
-        pass
+      super(BuyContract, self).__init__(vehicle, customer)
+      self.monthly_payments = monthly_payments
+      self.MONTH = monthly_payments
```

Comment:
you don't need `self.monthly_payments` and 'self.MONTH` ..they are both the same ?